deps/rabbitmq_shovel_management/src/rabbit_shovel_mgmt.hrl

```diff
@@ -0,0 +1,8 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% Copyright (c) 2007-2021 VMware, Inc. or its affiliates.  All rights reserved.
+%%
+
+-define(SHOVEL_CALLS_TIMEOUT_MS, 5000).
```

Comment:
5 seconds is an absolute minimum. Both in RabbitMQ itself and client libraries, 15s is the absolute minimum we would use, in particular when the value is a constant.